Duplicate PE2 declaration in STM32F722ZE

**Describe the bug**

As discussed in this [thread](https://github.com/stm32duino/Arduino_Core_STM32/commit/a855f1291c5e54daf7858e1b39a171677aed04ff#r133331760), there is a duplicate declaration of `PE_2` in [variant_NUCLEO_F722ZE.cpp](https://github.com/stm32duino/Arduino_Core_STM32/blob/main/variants/STM32F7xx/F722Z(C-E)T_F732ZET/variant_NUCLEO_F722ZE.cpp) and therefore of `PE2` as well in [variant_NUCLEO_F722ZE.h](https://github.com/stm32duino/Arduino_Core_STM32/blob/main/variants/STM32F7xx/F722Z(C-E)T_F732ZET/variant_NUCLEO_F722ZE.h).

**To Reproduce**

Simple PlatformIO project with the following `platformio.ini`:
```
[env:r0]
platform = ststm32@^17.0.0
board = nucleo_f722ze
framework = arduino
build_unflags =
  -Os
build_flags =
  -D PIO_FRAMEWORK_ARDUINO_NANOLIB_FLOAT_PRINTF
  -D SERIAL_RX_BUFFER_SIZE=256
  -D SERIAL_TX_BUFFER_SIZE=256
  -O0
upload_protocol = stlink
debug_tool = stlink
monitor_speed = 115200
```

**Screenshots**
```
Compiling .pio\build\r0\SrcWrapper\src\HAL\stm32yyxx_hal_adc.c.o
In file included from C:\Users\nicol\.platformio\packages\framework-arduinoststm32\cores\arduino/variant.h:15,
                 from C:\Users\nicol\.platformio\packages\framework-arduinoststm32\system\STM32F7xx/stm32f7xx_hal_conf.h:4,
                 from C:\Users\nicol\.platformio\packages\framework-arduinoststm32\system\Drivers\STM32F7xx_HAL_Driver\Inc/stm32f7xx_hal.h:29,
                 from C:\Users\nicol\.platformio\packages\framework-arduinoststm32\system\Drivers\STM32F7xx_HAL_Driver\Src/stm32f7xx_hal.c:36,
                 from C:\Users\nicol\.platformio\packages\framework-arduinoststm32\libraries\SrcWrapper\src\HAL\stm32yyxx_hal.c:18:
C:\Users\nicol\.platformio\packages\framework-arduinoststm32\variants\STM32F7xx\F722Z(C-E)T_F732ZET/variant_NUCLEO_F722ZE.h:74: warning: "PE2" redefined
   74 | #define PE2                     56
      |
C:\Users\nicol\.platformio\packages\framework-arduinoststm32\variants\STM32F7xx\F722Z(C-E)T_F732ZET/variant_NUCLEO_F722ZE.h:49: note: this is the location of the previous definition
   49 | #define PE2                     31
      |
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - STM32 core version: 2.6.0

**Board (please complete the following information):**
 - Name: Nucleo F22ZE

**Additional context**
PlatformIO reports the following configuration:
```
CONFIGURATION: https://docs.platformio.org/page/boards/ststm32/nucleo_f722ze.html
PLATFORM: ST STM32 (17.0.0) > ST Nucleo F722ZE
HARDWARE: STM32F722ZET6 216MHz, 256KB RAM, 512KB Flash
DEBUG: Current (stlink) On-board (stlink) External (blackmagic, cmsis-dap, jlink)
PACKAGES:
 - framework-arduinoststm32 @ 4.20600.231001 (2.6.0)
 - framework-cmsis @ 2.50700.210515 (5.7.0)
 - toolchain-gccarmnoneeabi @ 1.120301.0 (12.3.1)
```

--- a/variants/STM32F7xx/F722Z(C-E)T_F732ZET/variant_NUCLEO_F722ZE.h
+++ b/variants/STM32F7xx/F722Z(C-E)T_F732ZET/variant_NUCLEO_F722ZE.h
@@ -71,7 +71,7 @@
 #define PD5                     53
 #define PD4                     54
 #define PD3                     55
-#define PE2                     56
+// 56 is PE2                   (31)
 #define PE4                     57
 #define PE5                     58
 #define PE6                     59

